Checkout v201 docs (#398)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/checkout/tutorials/buy-online-pickup-in-store.mdx

The following steps describe how to modify the [commerce-checkout.js](https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/blocks/commerce-checkout/commerce-checkout.js) block file in the boilerplate template to allow users to choose between delivery and in-store pickup during the checkout process.

<Aside type="note">

> The _**commerce-checkout.js**_ block is the only one that is fully functional and up-to-date with the latest version of the checkout drop-in component.

Reviewer: ```suggestion
The `commerce-checkout.js` block is the only fully functional implementation that's up-to-date with the latest checkout drop-in component.
```

> Use the following guidelines as a reference when creating a new checkout experience.

Reviewer: ```suggestion
Use these steps as a reference when creating your own checkout experience.
```
